package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "redux-saga-firebase",
-  "version": "0.0.9",
+  "version": "1.0.0",
```

Comment:
Please don't change it for now, I'll take update the version later before publishing. :)